### DISCUSSION TITLE
Much thanks + Questions on the future of the project.

### DISCUSSION THREAD

**Discussion by naturale0** [af/hyperpaper-planner#6] [General]
First of all, I want to thank you for sharing this. It has all the features I wanted and has nothing that I did not.

I have two questions:
1. Are you planning to make a 2022 version?
2. By any chance, are you planning to make the project open-sourced?

**Comment by af** [comment]
2022 version: yes, definitely!
Open source: no plans at this point, but I wouldn't rule it out entirely
